lib/src/screens/root/root.dart

```diff
@@ -28,15 +30,18 @@ class RootState extends State<Root> with WidgetsBindingObserver {
   RootState()
     : _isInactiveController = StreamController<bool>.broadcast(),
     _isInactive = false,
+   _requestAuth = getIt.get<AuthService>().requireAuth(),
```

Comment:
Better to avoid directly calls of getIt inside of your users classes. Better to make "factory" callback property which will return AuthService and then call requiredAuth on that or directly callback which will call that.